internal/provider/resource_consul_cluster_test.go

```diff
@@ -193,7 +243,11 @@ func TestAccConsulCluster(t *testing.T) {
 				Config: testConfig(setTestAccConsulClusterConfig(updatedConsulCluster)),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckConsulClusterExists(resourceName),
-					resource.TestCheckResourceAttr(resourceName, "size", "SMALL"),
+					resource.TestCheckResourceAttr(resourceName, "size", "MEDIUM"),
```

Comment:
I had to change to a `STANDARD`. This update test was failing with the following error:

```
 Error: error updating Consul cluster (test-202302131304): [PATCH /consul/2021-02-04/organizations/{cluster.location.organization_id}/projects/{cluster.location.project_id}/clusters/{cluster.id}][400] Update default  &{Code:3 Details:[] Error:Development tier clusters only have one size and cannot be changed. Message:Development tier clusters only have one size and cannot be changed.}
 ```
 
 Development clusters cannot change sizes